docker-images/cold-backup/Makefile

```diff
@@ -0,0 +1,9 @@
+PROJECT_NAME=cold-backup
```

Comment:
If we really want to have this in this repository, it needs to be plugged into the build - probably by adding it here: https://github.com/strimzi/strimzi-kafka-operator/blob/37bbc7181abe3096daf8490c67e2912fa2d3d5f8/docker-images/build.sh#L12